### PR TITLE
goals add words and grey lines to add new goal

### PR DESCRIPTION
**Pull request by MollieCourtenay**

### CODE REVIEW

File: web/templates/goal/form.html.eex [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+      <strong>Time limited</strong>, set a reasonable time frame to reach your goal.<br> </p>
+<p class="tl">Describe a personal goal.</p>
+  
   <%= form_for @changeset, @action, fn f -> %>
       <div class=" mb3 pb3 bb b--hl-grey bw2">
         <%= textarea f, :content, placeholder: "Content", class: "db border-box hover-black w-100 ba b--black-20 pa2 br2 mb2 default-font" %>
       </div>
       <p class="b pt2 mb0">Steps to take:</p>
       <p class="mt1">List the things you could do to achieve this goal.</p>
+        <p>Come back each and tick each step off as you complete them.</p>
```
Comment (katbow) [inline web/templates/goal/form.html.eex]: Typo? Think it should say "Come back ~each~ and tick each step off as you complete them."

File: web/templates/goal/form.html.eex [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
       <%= inputs_for f, :steps, fn s -> %>
         <div class="mb3">
           <%= checkbox s, :complete, class: "checkbox" %>
           <%= text_input s, :details, class: "ph3 input-reset bn hl-bg-grey w-90 " <>
             Healthlocker.ComponentView.highlight_errors(@changeset, :details) %>
         </div>
       <% end %>
-
+ <div class=" mb3 pb3 bb b--hl-grey bw2">
```
Comment (katbow) [inline web/templates/goal/form.html.eex]: just need the closing tag with this as well like how you did in edit. 

`<div class=" mb3 pb3 bb b--hl-grey bw2"></div>`

File: web/templates/goal/form.html.eex [hunk 3]
```diff
@@ -24,6 +33,7 @@
         <% end %>
         <p class="i">Important goals will be at the top of the list.</p>
       </div>
+  <div class=" mb3 pb3 bb b--hl-grey bw2">
```
Comment (katbow) [inline web/templates/goal/form.html.eex]: Same as above `<div class=" mb3 pb3 bb b--hl-grey bw2"></div>`

File: web/templates/goal/form.html.eex (hunk 1 above)
Comment (MollieCourtenay) [reply web/templates/goal/form.html.eex]: thanks!

**Comment by MollieCourtenay** [comment]
@katbow done those thanks

**Pull request merged**